<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>561</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>读书笔记</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/icon/42424.jpg</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget#widget
{
	background-color:#4f535e;
}

QWidget#widget_toolbar
{
	background-color:#cccccc;
}
QTreeWidget:treeWidget
{
   color:rgb(255, 255, 255);
}
QLabel
{
	font: 15pt &quot;微软雅黑&quot;;
}

QPushButton
{
	font: 14pt &quot;微软雅黑&quot;;
	border: 0px;
	
}
QToolButton
{
font: 14pt &quot;微软雅黑&quot;;
	border: 0px;
}


QTreeWidget#treeWidget
{
	background-color:#4f535e;
	border: 0px;
	font: 15pt &quot;微软雅黑&quot;;
	image: url(:/icon/search.jpg);
}

QWidget#widget_workspace
{
	background-color:#4f535e;
}

QWidget#widget_2
{
	background-color:#eeeeee;
}

QWidget#widget_3
{
	background-color:#eeeeee;
}
QTreeView::branch:has-children{
	
	image: url(:/icon/search.jpg);
}


QScrollArea#three_scrollArea
{
	background-color: rgb(255, 255, 255);
}

QLineEdit 
{
     border: 1px solid #cccccc;
     border-radius: 0px;
	 font: 15pt &quot;微软雅黑&quot;;
}

QComboBox
{
     border: 1px solid rgba(161, 161, 161);
     padding: 1px 2px 1px 2px;
	 font: 15pt &quot;微软雅黑&quot;;
}

QComboBox::drop-down
{
	subcontrol-origin: padding;
    subcontrol-position: top right;
	width: 20px;
    border-left-width:1px;
     border-left-color:  rgba(161, 161, 161);
 	 border-left-style: solid;
}

QComboBox::down-arrow
{
	image: url(:/icon/combox.gif);
}

QComboBox QAbstractItemView
{
	border: 1px solid rgba(161, 161, 161);
}

QTextEdit
{
	border: 1px solid #cccccc;
	font: 15pt &quot;微软雅黑&quot;;
}

QListWidget
{
	border: 1px solid #cccccc;
	font: 15pt &quot;微软雅黑&quot;;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>7</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_toolbar" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="horizontalSpacing">
           <number>5</number>
          </property>
          <item row="0" column="10">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <widget class="QToolButton" name="newfile">
            <property name="text">
             <string>新建笔记</string>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/icon/newfile.png</normalon>
             </iconset>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <widget class="QToolButton" name="importfile">
            <property name="text">
             <string>文件导入</string>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/icon/import.png</normalon>
             </iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QToolButton" name="delfile">
            <property name="text">
             <string>删除笔记</string>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/icon/delete.jpg</normalon>
             </iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QToolButton" name="toword">
            <property name="text">
             <string>导出为word</string>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/icon/output.jpg</normalon>
             </iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="7">
           <widget class="QToolButton" name="restorefile">
            <property name="text">
             <string>文件导出</string>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/icon/save.jpg</normalon>
             </iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="8">
           <widget class="QToolButton" name="categorys">
            <property name="text">
             <string>类别管理</string>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/icon/manager.jpg</normalon>
             </iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QToolButton" name="savefile">
            <property name="text">
             <string>保存笔记</string>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/icon/save.jpg</normalon>
             </iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_workspace" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QTreeWidget" name="treeWidget">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>15</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_6" native="true">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(:/icon/oo.png);</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="hh" native="true">
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">background-image: url(:/icon/bk.png);</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QWidget" name="widget_2" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>250</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>400</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <property name="spacing">
                    <number>20</number>
                   </property>
                   <item>
                    <widget class="QFrame" name="frame_4">
                     <property name="cursor">
                      <cursorShape>ArrowCursor</cursorShape>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color: rgb(255, 255, 255);
</string>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::Box</enum>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_9">
                      <property name="spacing">
                       <number>0</number>
                      </property>
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>0</number>
                      </property>
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                      <item>
                       <widget class="QLineEdit" name="searchword">
                        <property name="styleSheet">
                         <string notr="true">border:0px</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="ss">
                        <property name="maximumSize">
                         <size>
                          <width>38</width>
                          <height>38</height>
                         </size>
                        </property>
                        <property name="cursor">
                         <cursorShape>ArrowCursor</cursorShape>
                        </property>
                        <property name="styleSheet">
                         <string notr="true">border:0px</string>
                        </property>
                        <property name="text">
                         <string/>
                        </property>
                        <property name="icon">
                         <iconset>
                          <normalon>:/icon/search.png</normalon>
                         </iconset>
                        </property>
                        <property name="iconSize">
                         <size>
                          <width>38</width>
                          <height>38</height>
                         </size>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_2">
                     <item>
                      <widget class="QLabel" name="label">
                       <property name="text">
                        <string>笔记列表:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QListWidget" name="searchresults">
                       <property name="spacing">
                        <number>0</number>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_3" native="true">
                  <layout class="QVBoxLayout" name="verticalLayout_6">
                   <property name="spacing">
                    <number>10</number>
                   </property>
                   <property name="bottomMargin">
                    <number>9</number>
                   </property>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_5">
                     <item>
                      <widget class="QLabel" name="bookname">
                       <property name="text">
                        <string>书名</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="nameEd">
                       <property name="maximumSize">
                        <size>
                         <width>300</width>
                         <height>16777215</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_2">
                       <property name="text">
                        <string>作者</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="author"/>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_6">
                       <property name="text">
                        <string>出版社</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="place"/>
                     </item>
                     <item>
                      <spacer name="horizontalSpacer_5">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_8">
                     <item>
                      <widget class="QLabel" name="category">
                       <property name="text">
                        <string>类别</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QComboBox" name="categoryBox">
                       <property name="minimumSize">
                        <size>
                         <width>200</width>
                         <height>0</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_3">
                       <property name="text">
                        <string>出版时间</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="madetime"/>
                     </item>
                     <item>
                      <spacer name="horizontalSpacer_3">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_6">
                     <property name="spacing">
                      <number>7</number>
                     </property>
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_4">
                       <item>
                        <widget class="QLabel" name="label_4">
                         <property name="text">
                          <string>摘记</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <spacer name="verticalSpacer">
                         <property name="orientation">
                          <enum>Qt::Vertical</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>20</width>
                           <height>40</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <widget class="QTextEdit" name="note">
                       <property name="frameShadow">
                        <enum>QFrame::Plain</enum>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_7">
                     <property name="spacing">
                      <number>7</number>
                     </property>
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_5">
                       <property name="spacing">
                        <number>0</number>
                       </property>
                       <item>
                        <widget class="QLabel" name="label_5">
                         <property name="text">
                          <string>评注</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <spacer name="verticalSpacer_2">
                         <property name="orientation">
                          <enum>Qt::Vertical</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>20</width>
                           <height>40</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <widget class="QTextEdit" name="feeling">
                       <property name="frameShadow">
                        <enum>QFrame::Plain</enum>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_2">
                       <item>
                        <widget class="QLabel" name="label_cretetime">
                         <property name="text">
                          <string>创建时间：</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="createtime">
                         <property name="minimumSize">
                          <size>
                           <width>80</width>
                           <height>0</height>
                          </size>
                         </property>
                         <property name="text">
                          <string/>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <spacer name="horizontalSpacer">
                         <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_3">
                       <item>
                        <widget class="QLabel" name="label_updatetime">
                         <property name="text">
                          <string>更新时间：</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="updatetime">
                         <property name="minimumSize">
                          <size>
                           <width>80</width>
                           <height>0</height>
                          </size>
                         </property>
                         <property name="text">
                          <string/>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <spacer name="horizontalSpacer_2">
                         <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
